File: src/tests/Markify.Fixtures/Projects/Source/Struct/AccessModifier.cs
public struct PublicStruct { }

internal struct InternalStruct { }

public struct PrivateFoo
{
    private struct PrivateStruct { }
}

public class ProtectedFoo
{
    protected struct ProtectedStruct { }
}

public class ProtectedInternalFoo
{
    protected internal struct ProtectedInternalStruct { }
}

File: src/tests/Markify.Fixtures/Projects/Source/Struct/AccessModifier.vb
Public Structure PublicStruct
End Structure

Friend Structure InternalStruct
End Structure

Public Structure PrivateFoo
    Private Structure PrivateStruct
    End Structure
End Structure

Public Class ProtectedFoo
    Protected Structure ProtectedStruct
    End Structure
End Class

Public Class ProtectedInternalFoo
    Protected Friend Structure ProtectedInternalStruct
    End Structure
End Class